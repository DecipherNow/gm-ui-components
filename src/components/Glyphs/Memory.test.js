import React from "react";
import { shallow } from "enzyme";
import Memory from "./Memory";

describe("Memory", () => {
  it("matches snapshot", () => {
    const aMemory = shallow(<Memory />);
    expect(aMemory).toMatchSnapshot();
  });
});
