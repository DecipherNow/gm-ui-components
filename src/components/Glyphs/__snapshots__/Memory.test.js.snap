// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Memory matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Memory />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "ariaLabelledby": "ariaLabelledby",
      "backgroundColor": "currentColor",
      "backgroundOpacity": 1,
      "backgroundSizeRatio": 1,
      "borderColor": "currentColor",
      "borderOpacity": 1,
      "borderWidth": 1,
      "children": <path
        d="M17,15 L15,15 L15,16 L14,16 L14,15 L13,15 L13,16 L12,16 L12,15 L11,15 L11,16 L10,16 L10,15 L9,15 L9,16 L8,16 L8,15 L7,15 L7,16 L6,16 L6,15 L5,15 L5,16 L4,16 L4,15 L3,15 C2.44771525,15 2,14.5522847 2,14 L2,8 C2,7.44771525 2.44771525,7 3,7 L21,7 C21.5522847,7 22,7.44771525 22,8 L22,14 C22,14.5522847 21.5522847,15 21,15 L20,15 L20,16 L19,16 L19,15 L18,15 L18,16 L17,16 L17,15 Z M3,8 L3,14 L21,14 L21,8 L3,8 Z"
        id="Combined-Shape"
/>,
      "glyph": "",
      "glyphColor": "currentColor",
      "glyphSizeRatio": 1,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "d": "M17,15 L15,15 L15,16 L14,16 L14,15 L13,15 L13,16 L12,16 L12,15 L11,15 L11,16 L10,16 L10,15 L9,15 L9,16 L8,16 L8,15 L7,15 L7,16 L6,16 L6,15 L5,15 L5,16 L4,16 L4,15 L3,15 C2.44771525,15 2,14.5522847 2,14 L2,8 C2,7.44771525 2.44771525,7 3,7 L21,7 C21.5522847,7 22,7.44771525 22,8 L22,14 C22,14.5522847 21.5522847,15 21,15 L20,15 L20,16 L19,16 L19,15 L18,15 L18,16 L17,16 L17,15 Z M3,8 L3,14 L21,14 L21,8 L3,8 Z",
        "id": "Combined-Shape",
      },
      "ref": null,
      "rendered": null,
      "type": "path",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "ariaLabelledby": "ariaLabelledby",
        "backgroundColor": "currentColor",
        "backgroundOpacity": 1,
        "backgroundSizeRatio": 1,
        "borderColor": "currentColor",
        "borderOpacity": 1,
        "borderWidth": 1,
        "children": <path
          d="M17,15 L15,15 L15,16 L14,16 L14,15 L13,15 L13,16 L12,16 L12,15 L11,15 L11,16 L10,16 L10,15 L9,15 L9,16 L8,16 L8,15 L7,15 L7,16 L6,16 L6,15 L5,15 L5,16 L4,16 L4,15 L3,15 C2.44771525,15 2,14.5522847 2,14 L2,8 C2,7.44771525 2.44771525,7 3,7 L21,7 C21.5522847,7 22,7.44771525 22,8 L22,14 C22,14.5522847 21.5522847,15 21,15 L20,15 L20,16 L19,16 L19,15 L18,15 L18,16 L17,16 L17,15 Z M3,8 L3,14 L21,14 L21,8 L3,8 Z"
          id="Combined-Shape"
/>,
        "glyph": "",
        "glyphColor": "currentColor",
        "glyphSizeRatio": 1,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "d": "M17,15 L15,15 L15,16 L14,16 L14,15 L13,15 L13,16 L12,16 L12,15 L11,15 L11,16 L10,16 L10,15 L9,15 L9,16 L8,16 L8,15 L7,15 L7,16 L6,16 L6,15 L5,15 L5,16 L4,16 L4,15 L3,15 C2.44771525,15 2,14.5522847 2,14 L2,8 C2,7.44771525 2.44771525,7 3,7 L21,7 C21.5522847,7 22,7.44771525 22,8 L22,14 C22,14.5522847 21.5522847,15 21,15 L20,15 L20,16 L19,16 L19,15 L18,15 L18,16 L17,16 L17,15 Z M3,8 L3,14 L21,14 L21,8 L3,8 Z",
          "id": "Combined-Shape",
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
